File: src/com/pythonForDSA/python_dsa/recurrsion/backtracking/maze_2.js
const get_paths = (row, col, up) => {
    if(row == 1 && col == 1){
        ds.push(up);
        return;
    }
    if(row > 1){
        get_paths(row - 1, col, up + "D");  // go down
    }
    if(col > 1){
        get_paths(row, col - 1, up + "R");  // go up
    }
}

const ds = [];
get_paths(3, 3, "");
console.log(ds);

File: src/com/pythonForDSA/python_dsa/recurrsion/backtracking/maze_2.py
def print_path(up, row, col):
    if row == 1 and col == 1:
        ds.append(up)
        return
        
    if row > 1: 
        print_path(up + "D", row - 1, col)  # go down
    if col > 1:
        print_path(up + "R", row, col - 1)  # go up

ds = []
print_path("", 3, 3)
print(ds)